crypto/keyring/keyring_test.go

```diff
@@ -1162,13 +1356,13 @@ func TestRenameKey(t *testing.T) {
 			name: "rename a key",
 			run: func(kr Keyring) {
 				oldKeyUID, newKeyUID := "old", "new"
-				oldKeyInfo := newKeyInfo(t, kr, oldKeyUID)
+				oldKeyRecord := newKeyRecord(t, kr, oldKeyUID)
 				err := kr.Rename(oldKeyUID, newKeyUID) // rename from "old" to "new"
 				require.NoError(t, err)
-				newInfo, err := kr.Key(newKeyUID) // new key should be in keyring
+				newRecord, err := kr.Key(newKeyUID) // new key should be in keyring
 				require.NoError(t, err)
-				requireEqualRenamedKey(t, newInfo, oldKeyInfo) // oldinfo and newinfo should be the same
-				oldKeyInfo, err = kr.Key(oldKeyUID)            // old key should be gone from keyring
+				requireEqualRecord(t, newRecord, oldKeyRecord, false) // oldKeyRecord and newRecord should be the same except name
```

Comment:
After running `kr.Rename(oldKeyUID, newKeyUID)` , all fields of `newRecord` and `oldKeyRecord` should match except names (`newRecord.Name = "new"`, `oldKeyRecord.Name = "old"`). Are you okay with that, @robert-zaremba ?

Reply:
why not using the `requireEqualRenamedKey`?

Reply:
OK